SimpleInventoryManagementSystem/IDatabase.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace SimpleInventoryManagementSystem
+{
+    public interface IDatabase
```

Comment:
the classes in Data Layer offten we naming them with Repository Suffix, for example: IProductRepository